multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -23,6 +23,9 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
```

Comment:
Yeah, turns out files override regular, so writing a new file and using it would work, but that won't be as simple as just overriding execstart.
I'm fine with either approach now.